### PR TITLE
[nnstreamer/apptest] script to build examples

### PR DESCRIPTION
**Pull request by jaeyun-jung**
To run app-test in nnstreamer repo, clone nnstreamer-examples and install the examples.

**Changelogs**
* Version 2:
  - update the description

* Version 1:
  - add script to build the nnstreamer examples

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh [hunk 1]
```diff
@@ -117,7 +117,17 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     # Install a nnstreamer library and examples
     ninja -C build install
 
+    # Clone nnstreamer examples repository
```
Comment (leemgs) [inline ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: It is my trivial proposal. :)
* before: # Clone nnstreamer examples repository
* after: # Clone the nnstreamer-example repository

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh [hunk 2]
```diff
@@ -117,7 +117,17 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
...
+    git clone https://github.com/nnsuite/nnstreamer-example.git exam-tmp
+    cd exam-tmp
+
+    # Build and install nnstreamer examples
+    meson --prefix=${NNST_ROOT} --sysconfdir=${NNST_ROOT} --libdir=lib --bindir=bin --includedir=include build
```
Comment (leemgs) [inline ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: :100:

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh [hunk 3]
```diff
@@ -114,10 +114,20 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     # Build a source code
     meson --prefix=${NNST_ROOT} --sysconfdir=${NNST_ROOT} --libdir=lib --bindir=bin --includedir=include build
 
-    # Install a nnstreamer library and examples
+    # Install a nnstreamer library
```
Comment (jaeyun-jung) [inline ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: With these meson options, it does not install the examples (custom filters in nnstreamer reposotiry)

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh (hunk 1 above)
Comment (jaeyun-jung) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: I updated this.

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh (hunk 3 above)
Comment (leemgs) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: Acked.

**Pull request merged**